templates/base.html

```diff
@@ -171,6 +171,17 @@
 
 
     {% autoescape off %}
+
```

Comment:
I have some concerns about the way are are passing the data between the server and client here, and that it doesnt match our latest practices, but this should work, at least initially.